### DISCUSSION TITLE
Cutscenes audio/video format shift?

### DISCUSSION THREAD

**Discussion by tamlin-mike** [DescentDevelopers/Descent3#35] [Libraries]
Regarding cutscenes ACM/MVE, a comment HN stated "FFMPEG seems to support these formats".

Should that turn out incorrect or less feasible, to keep any transcoding as portable as possible perhaps theora+vorbis would be the best/most portable/futureproof/unencumbered?

Just an idea.

**Comment by JeodC** [comment]
I initially wanted to push for theoradec to maintain compatibility with Ryan Gordon's port, but that will leave out people who own older Descent 3 versions like the CDs and the Loki x86 port. I think it's best to avoid asking people to purchase the game again in order to access game assets--and those who own Descent 3 on GOG/Steam are able to access mve assets as well as ogv assets. Ryan [recently](https://github.com/DescentDevelopers/Descent3/issues/9#issuecomment-2067831249) commented on the collab effort, by the way. #2 expands on potential avenues.
